source/retryable-writes/tests/unified/insertOne-serverErrors.yml

```diff
@@ -75,3 +75,122 @@ tests:
           - { _id: 1, x: 11 }
           - { _id: 2, x: 22 }
           - { _id: 3, x: 33 }  # The write was still applied
+
+  - description: "RetryableWriteError label is added based on top-level code in pre-4.4 server response"
+    runOnRequirements:
+      - minServerVersion: "4.2"
+        maxServerVersion: "4.2.99"
+        topologies: [ replicaset, sharded ]
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: *client0
+          failPoint:
+            configureFailPoint: failCommand
+            # Trigger the fail point twice to allow asserting the error label in
+            # the retry attempt's response.
```

Comment:
This initially tripped me up while implementing the libmongoc POC and diagnosing some CI failures. When we assert that the RetryableWriteError label is added, those tests will implicitly have a retry attempt, so we need to fail the second attempt to ensure the label is present for our assertion.

This differs from the mongos test asserting that the label is _not_ added. There, the driver makes no retry attempt so we need only fail once and inspect the `insertOne` result for an error w/o the label.